proxy/pom.xml

```diff
@@ -107,6 +102,16 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.33.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-daemon</groupId>
+      <artifactId>commons-daemon</artifactId>
+      <version>1.0.15</version>
```

Comment:
should we have the commons-daemon github not as a submodule but rather as a required dependency to build the project? We obviously depend on a bunch of tools as well as a compiled jdk, can we just assume you have some version of the commons-daemon native code in a local directory? (with some readme that details all the necessary dependencies and tooling needed to build the agent in a readme file of course)